src/pow_avx.c

```diff
@@ -573,6 +575,10 @@ bool PowAVX(void *pow_ctx)
         pthread_join(threads[i], NULL);
         if (pitem[i].n == -1)
             completedIndex = i;
+        if (pitem[i].ret >= 0)
```

Comment:
We can simplify it:
```
ctx->hash_count += (uint64_t) ((pitem[i].ret >= 0) ? pitem[i].ret : -pitem[i].ret + 1);
```

Reply:
Change in the commit : https://github.com/DLTcollab/dcurl/pull/73/commits/370382269cbd3084df6669a9f97c4f18875c0726